clippy_lints/src/doc/markdown.rs

```diff
@@ -117,6 +134,17 @@ fn check_word(cx: &LateContext<'_>, word: &str, span: Span, code_level: isize, b
         // try to get around the fact that `foo::bar` parses as a valid URL
         && !url.cannot_be_a_base()
     {
+        let Some(fragment_span) = fragments.span(cx, range.clone()) else {
+            return ControlFlow::Break(());
```

Comment:
This seems wrong. One spot failing to get a span doesn't mean all the others will.

Reply:
Fixed!